mapie/tests/test_classification.py

```diff
@@ -1768,9 +1750,8 @@ def test_get_true_label_cumsum_proba_shape() -> None:
         estimator=clf, random_state=random_state
     )
     mapie_clf.fit(X, y)
-    cumsum_proba, cutoff = mapie_clf._get_true_label_cumsum_proba(
-        y, y_pred
-    )
+    classes = mapie_clf.classes_
```

Comment:
Same : this test should go in an other file

Reply:
Done 👍